src/main/java/net/fabricmc/loom/api/LoomGradleExtensionAPI.java

```diff
@@ -211,4 +211,10 @@ default String getCustomManifest() {
 	 * @return the property controlling the transitive access wideners
 	 */
 	Property<Boolean> getEnableTransitiveAccessWideners();
+
+	Property<String> getIntermediaryUrl();
+
+	default void setIntermediaryUrl(String url) {
+		getIntermediaryUrl().set(url);
+	}
```

Comment:
This provides no benefit. You can just do `intermediaryUr = "https://example.com"` on the popertry.